src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1041,6 +1030,11 @@ FixedwingAttitudeControl::task_main()
 					_wheel_ctrl.reset_integrator();
 				}
 
+				float roll_sp = _att_sp.roll_body;
```

Comment:
It was you that added these variables before at this point so if you don't mind adding this comment, we appreciate.